.github/workflows/pr-commands.yml

```
@@ -11,6 +11,8 @@ jobs:
   main:
     permissions:
       contents: read
+      # The "id-token: write" permission is required by "get-vault-secrets" action. We request it here
```

```suggestion
      # The "pull-requests: write" permission is required by "get-vault-secrets" action. We request it here
```
(I'm assuming the yaml is correct and the comment is not)

---

Good catch, I forgot to add the actual permission for token 🤦🏻‍♀️